flow/designs/gf12/swerv_wrapper/config_hier.mk

```diff
@@ -16,7 +16,14 @@ export RTLMP_MIN_MACRO = 4
 export RTLMP_RPT_DIR = $(OBJECTS_DIR)/rtlmp
 export RTLMP_RPT_FILE = partition.txt
 export RTLMP_CONFIG_FILE = ./designs/$(PLATFORM)/$(DESIGN_NAME)/rtlmp_config.txt
-export RTLMP_BLOCKAGE_FILE = $(OBJECTS_DIR)/rtlmp/partition.txt.blockage
+#export RTLMP_BLOCKAGE_FILE = $(OBJECTS_DIR)/rtlmp/partition.txt.blockage
+export RTLMP_AREA_WT = 0.01
+export RTLMP_WIRELENGTH_WT = 54.7
+export RTLMP_OUTLINE_WT = 74.71
+export RTLMP_BOUNDARY_WT = 100.0
+export RTLMP_MACRO_BLOCKAGE_WT = 50.0
+export RTLMP_LOCATION_WT = 100.0
+export RTLMP_NOTCH_WT = 100.0
```

Comment:
these settings are removed from all config_files except one (gf12/swerv_wrapper). I have left the settings in one file just so that the variable names and user settings are tested and as an example